samples/testing-frameworks/serenity-junit/src/test/resources/serenity.conf

```diff
@@ -6,7 +6,7 @@ webdriver {
     browserVersion = "latest"
```

Comment:
The customer has problems choosing the OS, and they requested an example that lets them switch between Windows/Linux/macOS.
I can see we're supporting only Windows here. We need to add support for macOS and Linux, too.
Ideally the solution would require the customer only to run the test differently (eg. a command line option), and not to make any changes to the actual scripts.